tests/cypress/page-object/fields/choiceTreeField.ts

```diff
@@ -15,7 +15,7 @@ export class ChoiceTreeField extends Field {
     }
 
     closeTree(): ChoiceTreeField {
-        cy.get('body').click();
+        cy.get('body').click('topLeft');
```

Comment:
choicetree cypress test is failing here because it's trying to click in the middle of body by default and accidentally clicking on one of the options. fixing by clicking out of the way of elements.